platform/views/platformViews.groovy

```diff
@@ -56,3 +56,12 @@ listView("edx-platform-pipeline-pr-tests") {
     }
     columns DEFAULT_VIEW.call()
 }
+
+listView("edx-platform-private-tests") {
+    description('Jobs for running tests on edx-platform-private')
+
+    jobs {
+        regex("edx-platform-*_private")
```

Comment:
can't recall, do we have private hawthorn/ironwood jobs? if so, will they match this regex?